lib/razorpay/fund_account.rb

```diff
@@ -0,0 +1,20 @@
+require 'razorpay/request'
+require 'razorpay/entity'
+
+module Razorpay
+  # Order API allows you to associate
+  # Payments with an order entity
+  class FundAccount < Entity
```

Comment:
add class level comments related to FundAccount class

Reply:
done